### PR TITLE
Product card

### PR DESCRIPTION
**Pull request by davrivas**
That's all for today. I will prepare my demo.

### CODE REVIEW

File: static/css/products.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        margin-bottom: 20px;
+    }
+}
+
+/*  Tablet: */
+
+@media (min-width: 481px) and (max-width: 959px) {
+    .wrapper {
+    	max-width: 959px;
+        margin-top: 350px;
```
Comment (cavilarts) [inline static/css/products.css]: Look this indentation

File: static/css/products.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    }
+
+    .productList {
+        float: right;
+        width: 80%;
+    }
+
+	.productCard {
+        background-color: #F4F4F4;
+		display: inline-block;
```
Comment (cavilarts) [inline static/css/products.css]: Same problem here indentation

File: static/css/products.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+	position: relative;
+    margin: auto;
+    padding: auto;
+}
+
+.productCard {
+	width: 300px;
+	height: 200px;
+	border: 1px solid #000;
+    font-family: Tahoma;
```
Comment (cavilarts) [inline static/css/products.css]: Same problem here indentation

File: static/css/products.css [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.productCard:nth-child(1) {
```
Comment (cavilarts) [inline static/css/products.css]: if this is dynamic how i can add the images by styles?

Comment (davrivas) [inline static/css/products.css]: I don't know. I want you tell me it for learn how to do that.

Comment (davrivas) [inline static/css/products.css]: And no, that's not dynamic, I just added the images like this just for save time.

**Pull request merged**